IPs/RSKIP143.md

```diff
@@ -0,0 +1,42 @@
+# Segwit Compatibility
+
+|RSKIP          |143          |
+| :------------ |:-------------|
+|**Title**      |Segwit Compatibility|
+|**Created**    |22-OCT-19 |
+|**Author**     |PGP |
+|**Purpose**    |Sca,USa |
+|**Layer**      |Core |
+|**Complexity** |2 |
+|**Status**     |Draft|
+
+## Abstract
+
+This document describes the procedure allows the 2wp proccess to accept segwit transaction sent using a P2SH-PWPKH wallet. 
+
+## Motivation
+
+The 2wp proccess only allows transaction who has been sent using a P2PK address. This RSKIP describes how Bridge could accept P2SH-PWPKH[1] since this is one of the most popular wallet nowadays and the segwit transactions needs less space in a block. 
+
+## Specification
+
+During the execution of the process, it parse each new transaction sent to obtain public key from it. If this identification cannot be made, Bridge considers the transaction as invalid.
+The changes related to this RKSIP refer to changing the Bridge code of RegisterBtcTransactions function. This function is responsible to parse the transactions and obtaining the public key value of the data sent to it. 
+RegisterBtcTransactions performs a number of validations to the BTC transaction to verify it legitimacy. After those validations, it will also check if it's a lock, and in case it is, it will then perform the lock using the sender's public key. These last two steps need to be changed.
```

Comment:
`RegisterBtcTransactions` performs a number of validations to the BTC transaction to verify ~~it~~ **it's** legitimacy.

Reply:
After those validations **have passed,** it ~~will~~ also check **(s)** if it's a lock, ~~and in case it is,~~ in that case it ~~will then~~ perform **(s)** the lock using the sender's public key. These last two steps **are the ones that** need to be changed.